pkgs/by-name/ne/nemo-preview/package.nix

```diff
@@ -0,0 +1,69 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  meson,
+  pkg-config,
+  ninja,
+  glib,
+  gtk3,
+  cjs,
+  gtksourceview4,
+  gobject-introspection,
+  libmusicbrainz5,
+  webkitgtk_4_1,
+  clutter-gtk,
+  clutter-gst,
+  wrapGAppsHook3,
+  nemo,
+  xreader,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "nemo-preview";
+  version = "6.4.0";
+
+  src = fetchFromGitHub {
+    owner = "linuxmint";
+    repo = "nemo-extensions";
+    tag = finalAttrs.version;
+    hash = "sha256-39hWA4SNuEeaPA6D5mWMHjJDs4hYK7/ZdPkTyskvm5Y=";
+  };
+
+  sourceRoot = "${finalAttrs.src.name}/nemo-preview";
+
+  nativeBuildInputs = [
+    wrapGAppsHook3
+    gobject-introspection
+    meson
+    pkg-config
+    glib
+    ninja
+    xreader
```

Comment:
This likely belongs to `buildInputs`, meson looks for `xreader-document-1.5` and `xreader-view-1.5` from the host.

Reply:
Is that the only change needed? I just want to implement all the changes and get this thing merged.